config/jobs/kubernetes/sig-release/release-branch-jobs/1.19.yaml

```diff
@@ -695,7 +717,7 @@ periodics:
         value: "true"
       - name: BUILD_TYPE
         value: bazel
-      image: gcr.io/k8s-testimages/krte:v20210312-67f589a-1.19
+      image: gcr.io/k8s-testimages/krte:v20210311-6badf7a-1.19
```

Comment:
Unintended change?

Reply:
> Unintended change?

Unskillful rebase attempt ;) thanks for catching this. Let me fix that in a minute.

Reply:
Fixed. Thanks again for catching this!